utah-design-system-website/src/react/components/routing/Routing.jsx

```diff
@@ -74,7 +74,7 @@ export function Routing() {
                 content={page.content}
                 contentRef={contentRef}
                 // @ts-ignore
```

Comment:
the ts-ignore is for `menus={menuSecondary}`
> Type 'undefined' is not assignable to type 'WebsiteMainMenu'.

Changing the code to be `menus={menuSecondary ?? null}` fixes the typescript error and makes sure that an undefined isn't accidentally passed in to the props. If the VerticalMenu ever tests `menus === null` it would fail if `undefined` were passed in.